### DISCUSSION TITLE
repaint placed scenery

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18639] [Feature requests] [answered]
Id like a paintbrush tool similar to the one used for alternative track colors except in the scenery menu as a way to change the color of scenery after it has been placed

**Comment by fidwell** [comment, marked as answer]
this thing?
![image](https://user-images.githubusercontent.com/5436387/203660799-3cab6f46-619e-4a1f-8265-a43e23c82f13.png)

**Reply by 733737** [reply]
I want the ability to change the color of scenery placed previously

**Reply by fidwell** [reply]
that's exactly what it does

**Reply by 733737** [reply]
really?

**Reply by fidwell** [reply]
try it
